.github/workflow_templates/e2e.abort.multi.yml

```diff
@@ -0,0 +1,121 @@
+{!{/*
+Multifile generator of e2e workflows.
```

Comment:
Some thoughts for e2e-clean workflow.

- The workflow is named 'e2e-clean' and command is 'e2e-abort'. Just a note: 'e2e-abort.yaml' name will be sorting-friendly =)
- There are 8 arguments for /e2e/abort command. We've discussed it can be only 3:
  `/e2e/abort PR_NUM RUN_ID E2E_OPTS`
  Other options can be calculated.
- There are multiple e2e workflows to distinguish workflows in UI. I think there is no need to distinguish providers for 'e2e clean', it can be one workflow. Also provider and layout are in E2E_OPTS.
- There is no way to clean multiple clusters in one run, so one clean job is sufficient, no need to render multiple jobs for each cri-version pair.

Reply:
1. done
2. unfortunately we need another arguments for it. I left comment [for describe](https://github.com/deckhouse/deckhouse/pull/2534/files#diff-d52c4bc185509431011a5d722fa9687396b1f3083fa3a388065fbf88c7605246R34) my decision

3-4. I had some troubles with it and i spent a lot of time for it. I think, we rebuild ci in next epic and we able to do it right.

Reply:
Oh, I've got it: there are difference in secrets for e2e script. Let it be multi template.